json-spec/json-schema/stac-item.json

```diff
@@ -49,41 +49,48 @@
               "title": "Item links",
               "description": "Links to item relations",
               "type": "object",
-              "properties": {
-                "self": {
-                  "allOf": [
-                    {
-                      "$ref": "#/definitions/link"
-                    },
-                    {
-                      "properties": {
-                        "rel": {
-                          "type": "string",
-                          "pattern": "^self$"
-                        }
-                      }
-                    }
-                  ]
-                }
+              "items": {
+                "$ref": "#/definitions/link"
               },
-              "patternProperties": {
-                ".+": {
-                  "$ref": "#/definitions/link"
```

Comment:
@francbartoli - did you mean to delete these? and the "pattern" stuff above? I'm pretty sure it's me messing this up - I accidentally merged it to master when I wanted to on dev, and then tried to revert, and it messed everything up. If you can revert this commit in your branch that'd be great. And I'll try to fix it on master, where it seems to still be messed up.